### PR TITLE
Fix ACE_TSS usage.

### PR DESCRIPTION
**Pull request by H0zen**
- This fixes the infamous "ACE_Thread::setspecific() failed!: No error"
    at servers' exit on Windows builds.

### CODE REVIEW

File: src/shared/Database/Database.cpp [hunk 1]
```diff
@@ -46,8 +46,8 @@ struct DBVersion
 
 const DBVersion databaseVersions[COUNT_DATABASES] = {
     { "World", WORLD_DB_VERSION_NR, WORLD_DB_STRUCTURE_NR, WORLD_DB_CONTENT_NR, WORLD_DB_UPDATE_DESCRIPTION }, // DATABASE_WORLD
-    { "Realmd", REALMD_DB_VERSION_NR, REALMD_DB_STRUCTURE_NR, REALMD_DB_CONTENT_NR, WORLD_DB_UPDATE_DESCRIPTION }, // DATABASE_REALMD
-    { "Character", CHAR_DB_VERSION_NR, CHAR_DB_STRUCTURE_NR, CHAR_DB_CONTENT_NR, WORLD_DB_UPDATE_DESCRIPTION }, // DATABASE_CHARACTER
+    { "Realmd", REALMD_DB_VERSION_NR, REALMD_DB_STRUCTURE_NR, REALMD_DB_CONTENT_NR, REALMD_DB_UPDATE_DESCRIPTION }, // DATABASE_REALMD
```
Comment (H0zen) [inline src/shared/Database/Database.cpp]: @billy1arm At erroneus startup the servers wrongly inform about content description of realmd and characters databases

**Comment by ryancheung** [comment]
Is this going to fix the "Dead Reference" error on server exit in Linux?

**Comment by H0zen** [comment]
The dead reference error appears at program exit, when one singleton depends on another which has been destroyed. Though I tested on freebsd and windoz, I cannot tell if this will fix your error (and I'm 90% sure it won't). Make a build and try. What linux distro do you use?

**Comment by ryancheung** [comment]
@H0zen Ubuntu 14.04 LTS 64bit

**Pull request merged**